code/DebugBar.php

```diff
@@ -201,7 +201,7 @@ public static function includeRequirements()
         }
         // If jQuery is already included, set to false
         $js = Requirements::backend()->getJavascript();
-        foreach ($js as $url) {
+        foreach ($js as $url => $args) {
             $name = basename($url);
```

Comment:
Requirements provide the path as the key and an array of arguments as the values in SS4. See [here](https://github.com/silverstripe/silverstripe-framework/blob/4/src/View/Requirements_Backend.php#L432-L436).